### ISSUE TITLE
regression 'missing checked class on toggles'

### ISSUE THREAD

**Issue by deneuxa** [Semantic-Org/Semantic-UI#3531]
same problem as here : https://github.com/Semantic-Org/Semantic-UI/issues/2506

**Comment by mh-cbon** [comment]
hi,

i think i m facing this issue. I did not build semantic from scratch, i downloaded pre compiled packages from the website (https://github.com/Semantic-Org/UI-Form/archive/master.zip && https://github.com/Semantic-Org/Semantic-UI-CSS/archive/master.zip).

I have html like this,

```html
  <div class="ui form">
    <div class="inline field">
      <div class="ui toggle checkbox start">
        <input tabindex="0" class="hidden" type="checkbox">
        <label>Start</label>
      </div>
...
```

When i add the `checked` class to `.start`, it won t animate the toggler.

I searched for the .checked selector in the css files and could not find much. Only that rule,
```css
i.icon.checked.calendar:before {
  content: "\f274";
}
```

@jlukic 

should i go for building my own semantic via npm packages ?

this bug seems different from aforementioned one.
